courses/models_test.py

```diff
@@ -91,12 +92,34 @@ def test_program_page_properties():
     program = ProgramFactory.create()
     assert program.description is None
     assert program.duration is None
-    assert program.outcomes is None
-    program_page = ProgramPageFactory.create(
-        program=program, description="<p>desc</p>", duration="1 week"
+    ProgramPageFactory.create(
+        program=program,
+        title="<p>page title</p>",
+        subhead="subhead",
+        description="<p>desc</p>",
+        duration="1 week",
+        video_title="<p>title</p>",
```

Comment:
why `html` is embedded here in the fields?

Reply:
This is a rich text field implemented in #140